### PR TITLE
[roseus_smach] add async join state

### PR DESCRIPTION
**Pull request by furushchev**

### CODE REVIEW

**Comment by garaemon** [comment]
merge origin/master

**Comment by furushchev** [comment]
Thanks to you, travis passed. Please merge!

File: roseus_smach/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,16 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(roseus_smach)
 
-find_package(catkin REQUIRED COMPONENTS euslisp roseus smach smach_ros smach_msgs)
+find_package(catkin REQUIRED COMPONENTS euslisp std_msgs actionlib roseus smach smach_ros smach_msgs message_generation)
+
+add_action_files(
+  DIRECTORY action
+  FILES
+  Multiply10.action
```
Comment (k-okada) [inline roseus_smach/CMakeLists.txt]: Is this file used for roseus_smach or just for the test?

**Comment by furushchev** [comment]
@k-okada sorry for late response.
This is just for test.

**Pull request merged**